### DISCUSSION TITLE
is Pathways or Metaversity Map Available?

### DISCUSSION THREAD

**Discussion by russelpearl** [learningeconomy/LearnCard#327] [Help] [answered]
Hi. I wanted to ask if the Pathways or Metavesity Map available on the sample components? 
I tried but failed to find it.

**Comment by Custard7** [comment, marked as answer]
> Hi. I wanted to ask if the Pathways or Metavesity Map available on the sample components? 
> I tried but failed to find it. 

Hey @russelpearl 👋

Unfortunately the pathways and map components are not plug and play drop-in components yet, but are built using everything in the SDK.

What is your MVP that you are trying to build, and what technical problems are you hoping to solve with the SDK? 

That would help me respond with specific steps you can take! 👍🤠
